app/actions/views/post.js

```diff
@@ -37,3 +39,16 @@ export function sendAddToChannelEphemeralPost(user, addedUsername, message, chan
         });
     };
 }
+
+export function setMenuActionSelector(dataSource, onSelect, options) {
```

Comment:
Two things could be improved here:
1. The async isn't needed
2. Since this is only dispatching a single action and isn't using getState, we can just return the action directly instead of returning a function
```javascript
export function setMenuActionSelector(...) {
    return {
        type: ViewTypes.SELECTED_ACTION_MENU,
        data: {...},
    };
}
```

Reply:
Good point, will update